### ISSUE TITLE
Docstring for VSCode preview

### ISSUE THREAD

**Issue by stefan-balke** [librosa/librosa#1823]
Hi,

I just stumbled on the Docstring preview in VSCode and that numpy's version is much nicer whereas librosa only shows the TOC.

Seems that VSCode simply renders the Docstring given in the `__init__.py`.
https://github.com/librosa/librosa/blob/main/librosa/__init__.py

It could simply be a copy-pasta from the docs starting page stating what librosa is about.

![image](https://github.com/librosa/librosa/assets/7326732/11711322-4ea9-403e-b57c-a63685577254)
![image](https://github.com/librosa/librosa/assets/7326732/ff23d182-b531-4286-8e2d-16a19049bb88)

**Comment by NotNotoginseng** [comment]
Is this issue still open? I'd like to contribute.

**Comment by bmcfee** [comment]
@NotNotoginseng yes, please go for it!  At this point though, I'll ask that you base a change like this on the `1.0.0dev` branch rather than `main`, as quite a few things have shifted around and not yet made it back into the main branch.

**Comment by NotNotoginseng** [comment]
@bmcfee Please let me know if there is anything else you'd like to see in this, otherwise I made it more visually appealing (imo).
